### DISCUSSION TITLE
Existing containers to a new pod

### DISCUSSION THREAD

**Discussion by Puchen72** [containers/podman#21557] [Q&A]
Hi. I have a problem. 

I have installed a few containers which should communicate with each others. After some research I found it better to place them in a POD. 
Now to my question: Can I copy/move theese containers into at new pod? If that is the case how?

When I started the containers I have now I ran the following command:
curl --proto '=https' --tlsv1.2 -O https://raw.githubusercontent.com/NUKIB/misp/main/docker-compose.yml
docker compose up -d
How can I get the command above in a pod?

Regards 
Thom

**Comment by rhatdan** [comment]
You can not get existing containers into a Pod. A pod has to be configured first.  And then the containers added to a pod.

**Reply by rhatdan** [reply]
You can add an remove containers from different network stacks.

**Reply by Puchen72** [reply]
So I should be able to create a pod than get the command that I wrote into a pod?

**Reply by rhatdan** [reply]
You create a pod and then add containers to the pod, each containers can have a command.
